Orm/Xtensive.Orm/Orm/Logging/IndentManager.cs

```diff
@@ -26,35 +26,35 @@ public void Dispose()
           return;
         disposed = true;
         CurrentIndentValueAsync.Value = oldIndent;
-        if (endAction!=null)
-          endAction.Invoke();
+        endAction?.Invoke();
       }
 
-      public IndentScope(string oldIndent, Action endAction)
+      public IndentScope(int oldIndent, Action endAction)
       {
         this.oldIndent = oldIndent;
         this.endAction = endAction;
+        disposed = false;
       }
     }
 
-    private const string SingleIndent = "  ";
+    private const int SingleIndent = 2;
```

Comment:
I think it will be right to rename it to SingleIndentLength

Reply:
renamed